### DISCUSSION TITLE
Resolution issues

### DISCUSSION THREAD

**Discussion by kiasho** [awawa-dev/HyperHDR#647] [General]
I am having some issues getting my devices to show the best resolution on my new tv. My setup is all devices go to my denon amp and the the amp goes to my capture card (Rybozen 4K Audio Video Capture Card, HDMI USB 3.0 Video Capture Device, Full HD 1080P for Game Recording, Live Streaming Broadcasting) hyperhdr works as it should but i cannot get my sky q box to set its resolution to uhd as it says the tv isnt compatible this however works fine when the sky q is connected directly to tv. My fire stick 4k max also says this hdmi port does not support hdcp2.2 so I'm guessing the usb capture card is causing issues with both devices if so and suggestions on a fix or a card that will allow devices to work at optimal resolution.

Thanks

**Comment by awawa-dev** [comment]
A rather typical symptom of using a USB grabber with a loop. Such built-in HDMI splitters are usually very limited, especially in cheap USB grabbers, and cause many side effects: limited resolution, color space, refresh rate and/or no HDR support, etc., even if the endpoint (TV) supports it. And even more expensive USB loop grabbers usually do not support HDCP. External HDMI splitter should solve most issues: search especially in pinned ms2130 thread.

**Reply by kiasho** [reply]
I do have an hdmi switch but was trying to to keep the electric usage to the minimum if possible however if i did choose to use it where in the setup would it need to go to solve the issues im having.

**Comment by kiasho** [comment]
I have added an ezcoo splitter which seems to have solved the resolution issues I have uhd 10bit on sky q and dolby vision on firestick but now the actual leds arent working with hyperhdr and im not sure why
